src/com/owncloud/android/lib/resources/activities/GetRemoteActivitiesOperation.java

```diff
@@ -116,7 +145,11 @@ protected RemoteOperationResult run(OwnCloudClient client) {
         return result;
     }
 
-    private ArrayList<Object> parseResult(String response) throws JSONException {
+    public Boolean hasMoreActivities() {
```

Comment:
Can we please use boolean instead of Boolean?

Reply:
Sure, dont know what the difference is

Reply:
Boolean has three states - Null, false and true. Since we don't need Null, boolean is preferred :) Thanks!